pyccel/stdlib/lists/lists.c

```diff
@@ -1,16 +1,5 @@
 #include "lists.h"
 
-size_t tSizes[8] = {
-            1,
-            sizeof(int8_t), 
-            sizeof(int16_t), 
-            sizeof(int32_t), 
-            sizeof(int64_t), 
-            sizeof(float), 
-            sizeof(double), 
-            sizeof(void *)
-};
-
 t_list   *allocate_list(size_t size, t_type type, void *elemnts) // va_arg could alow us to take in multiple list of elements
 {
     t_list *list;
```

Comment:
you shouldn't t change my code as I m still modifying it, it would create unnecessary conflicts, especially that you are not using them in your code.